coverage/base.css

```diff
@@ -172,16 +172,17 @@ table.coverage td span.cline-any {
 /* dark green */
 .status-line.high, .high .cover-fill { background:rgb(77,146,33) }
 .high .chart { border:1px solid rgb(77,146,33) }
-
-.medium .chart { border:1px solid #666; }
-.medium .cover-fill { background: #666; }
+/* dark yellow (gold) */
```

Comment:
Sorry, what are these for?

Reply:
I actually wanted to ask you about this... it looks like I somehow pulled in these changes when following the README instructions for coprocure-search-widget and components.coprocure.us. I wasn't sure whether these should be included or not, so I added them in the "update content from testing" diff so I can easily reset them.